mergeIntervals.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const merge = intervals => {
+  let len = intervals.length;
+  if (len <= 1) return intervals;
+  let res = [];
+  for (let i = 0; i < intervals.length; i++) {
+    for (let j = i + 1; j < intervals.length; j++) {
+      if (intervals[i][0] > intervals[j][0]) {
+        [intervals[i], intervals[j]] = [intervals[j], intervals[i]];
+      }
+    }
+  }
+
+  for (let i = 0; i < len - 1; i++) {
+    if (
```

Comment:
This solution works and that's fine. But if you're interested, I think we can make it simpler.
Try do it this way:
``
let res = [intervals.shift()];
for (let interval of intervals) {
...
}
```
I think there would less if-else branches.